system/Database/SQLSRV/Builder.php

```diff
@@ -290,6 +290,21 @@ private function getFullName(string $table): string
         }
 
         if ($this->db->escapeChar === '"') {
+            if (str_contains($table, '.')) {
```

Comment:
it may need to ensure "." is not in first and last char check as well so it always on between chars

Reply:
```suggestion
            if (str_contains($table, '.') && ! str_starts_with($table, '.') && ! str_ends_with($table, '.')) {
```

Reply:
Done.